dev/com.ibm.websphere.appserver.features/visibility/public/mpTelemetry-1.1/io.openliberty.mpTelemetry-1.1.feature

```diff
@@ -25,6 +25,7 @@ IBM-API-Package: \
   io.opentelemetry.semconv.resource.attributes;type="third-party",\
   io.opentelemetry.sdk.resources;type="third-party",\
   io.opentelemetry.instrumentation.annotations;type="third-party"
+  IBM-SPI-Package: io.openliberty.microprofile.telemetry.spi
```

Comment:
There is an unexpected leading space here.  Not sure if that ends up in the final feature or not, but it should be fixed before merging.